cola/models/main.py

```diff
@@ -467,6 +467,8 @@ def stage_paths(self, paths):
 
```

Comment:
When creating commits please sign off your work, per https://github.com/git-cola/git-cola/blob/master/CONTRIBUTING.md#sign-your-work

In the commit message editor we can use the `Ctrl+i` hotkey to add the Signed-off-by line.  You may need to adjust your config to include your full name.